src/openforms/registrations/contrib/objects_api/tests/test_update_payment_status_v2.py

```diff
@@ -152,13 +162,6 @@ def test_update_payment_status(self):
             "geometry_variable_key": "location",
         }
 
-        SubmissionPaymentFactory.create(
-            submission=submission,
-            status=PaymentStatus.completed,
-            amount=10.01,
-            public_order_id="TEST-123",
-        )
```

Comment:
why is this removed? the tests is specifically about payment status update, which is triggered when the payment is completed. The submissionpayment record being present *is* needed for a representative state